src/vm/prestub.cpp

```diff
@@ -269,6 +275,11 @@ PCODE MethodDesc::MakeJitWorker(COR_ILMETHOD_DECODER* ILHeader, CORJIT_FLAGS fla
          GetMethodTable()->GetDebugClassName(),
          m_pszDebugMethodName));
 
+#if defined(FEATURE_JIT_PITCHING)
+    bool fDoJitPitching = flags.IsSet(CORJIT_FLAGS::CORJIT_FLAG_DO_PITCHING);
+    CheckStacksAndPitch(fDoJitPitching);
```

Comment:
I was wondering if you needed any argument at all. For example imagine the code here was:

    CheckStacksAndPitch(TRUE);

Would that break the feature in some way? It wasn't clear to me what is being gained by identifying only the calls that originated in PreStubWorker rather than doing code pitching any time we JIT?